dev/components/good-map.html

```diff
@@ -1,6 +1,6 @@
 <script>
   'use strict';
-{
+var GoodMap = (() => {
```

Comment:
I'm also not an ES6/JS expert but I think in general avoiding `var` as much as possible is desirable. But perhaps this is ok given the somewhat unusual way this class is structured with `initCalled` and so forth.

The alternative I was considering is to revise line 19 below to explicitly state `class GoodMap extends HTMLElement`, but we'd still need somewhere for `callbackPromise` and `initCalled` in a safe namespace. I'm sure there are many ways to restructure this but all seem outside the scope of this change.

How about, can we just use `let` or `const` instead of `var` here on this line?

Reply:
I could have also refactored to make initCalled to be static members of GoodMap, but making them const would require fiddling with making object properties read only. Keeping a closure seemed a less invasive solution.

Changed to const.